upkie/config/spine_config.py

```diff
@@ -18,6 +18,21 @@
             "kp": 20.0,
             "kd": 1.0,
         },
+        "monitor": {
```

Comment:
I advocate against adding it to the default config (then it would run for all users regardless of whether they need it or not), but instead we could add an example so that users who need the feature can find it easily.